src/server/common/model/page/page-model.js

```diff
@@ -7,4 +10,40 @@ export class Page {
 
   /** @type {string} */
   sectionKey
+
+  /** @type {string} */
+  key
+
+  /** @type {string} */
+  view
+
+  /** @type {string} */
+  pageHeading
+
+  /** @type {string} */
+  pageTitle
+
+  /** @returns {string} */
+  get heading() {
+    return this.pageHeading
+  }
+
+  /** @returns {string} */
+  get title() {
+    return this.pageTitle
+  }
```

Comment:
We could remove these & set the properties to the names we want to access them under?

Reply:
to typescript complains as it is a getter somewhere (didnt want to dig around and risk breaking the wider app) and changing it to a property was what it didnt like, hence the slight doubling up